webxdc.d.ts

```diff
@@ -117,9 +117,15 @@ export interface Webxdc<StatusPayload> {
   /**
    * Webxdc are usually shared in a chat and run independently on each peer. To get a shared status, the peers use sendUpdate() to send updates to each other.
    * @param update status update to send
-   * @param description @deprecated, pass an empty string for backward compatibility
    */
-  sendUpdate(update: SendingStatusUpdate<StatusPayload>, description: ""): void;
+  sendUpdate(update: SendingStatusUpdate<StatusPayload>): void;
+  /**
+   * Deprecated, use {@link Webxdc.sendUpdate | sendUpdate(update)} instead
```

Comment:
hoping that seeing `sendUpdate(update)` here is clear enough for most readers. Wish using Erlang-style arity syntax (e.g. `sendUpdate/1`) was more common in web land 😄